app/controllers/FileChecksController.scala

```diff
@@ -35,10 +37,11 @@ class FileChecksController @Inject()(val controllerComponents: SecurityComponent
         fileChecks => Ok(views.html.fileChecksProgress(
           consignmentId,
           fileChecks.totalFiles,
```

Comment:
Now that we're passing more and more things to the view, I'm wondering if it would be easier to follow if we passed the `FileChecks` object itself, rather than passing all of its contents?

Reply:
Any thoughts on this one?

Reply:
Ah, yes, sorry, I forgot to reply. Yes, that is a good idea; I started working on it yesterday.